donkeycar/templates/complete.py

```diff
@@ -580,6 +586,10 @@ def run(self, mode, recording):
         inputs += ['pilot/angle', 'pilot/throttle']
         types += ['float', 'float']
 
+    if cfg.HAVE_PERFMON:
```

Comment:
Why not do this on the if-check on L#307 ?

Reply:
Because I am extending the `inputs` and `types` arrays which are initialized after L#562, those arrays are used fby TubHandler and MqttTelemetry. I can maybe move the initialization of Perfmon from lines 307 to here also if you think it'll make a difference.....

Reply:
I agree, it would be cleaner to have this in a single block here.